src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -3908,7 +4330,7 @@ pre_envp(char *const envp[], char ***new_envp)
 
 	for (j = 0; j < ARRAY_SIZE(env_list); j++) {
 		/* env is not set in environ. */
-		if (!env_set[i])
+		if (!env_set[j])
```

Comment:
Is this a bug-fix?

Reply:
Right. This is a bug-fix which was introduced in the exec() interception PR.